.travis.yml

```diff
@@ -22,7 +22,7 @@ script:
     if [[ $TRAVIS_PYTHON_VERSION != 3.8 ||
           ($TRAVIS_PYTHON_VERSION == 3.8 &&
            $GEOMSTATS_BACKEND != tensorflow) ]]; then
-      nose2 --with-coverage --verbose tests
+      nose2 --with-coverage tests
```

Comment:
you don't like the verbose option?

Reply:
It forces one to scroll down all the 700 lines of test to see the number of errors and the detailed traceback, which are both still available without the verbose mode.